taxi/tests/test_views.py

```diff
@@ -0,0 +1,111 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Driver, Car
+
+MANUFACTURER_URL = reverse("taxi:manufacturer-list")
+DRIVER_URL = reverse("taxi:driver-list")
+CAR_URL = reverse("taxi:car-list")
+
+
+class PublicTest(TestCase):
+    def test_login_required_for_manufacturer_list_page(self):
+        response = self.client.get(MANUFACTURER_URL)
+        self.assertNotEqual(response.status_code, 200)
+
+    def test_login_required_for_driver_list_page(self):
+        response = self.client.get(DRIVER_URL)
+        self.assertNotEqual(response.status_code, 200)
+
+    def test_login_required_for_car_list_page(self):
+        response = self.client.get(CAR_URL)
+        self.assertNotEqual(response.status_code, 200)
```

Comment:
The `assertNotEqual` method is used to check if the status code is not 200, which means any status code other than 200 will pass the test. It would be better to use `assertEqual` to check if the status code is 302, which is the standard response for a redirect. This way, you can be sure that the user is being redirected to the login page, which is the expected behavior.